subsys/bluetooth/controller/hal/nrf5/radio/radio.c

```diff
@@ -404,17 +409,14 @@ static void sw_switch(u8_t dir, u8_t phy_curr, u8_t flags_curr, u8_t phy_next,
 			    HAL_SW_SWITCH_TIMER_S8_DISABLE_PPI_ENABLE;
 		} else {
 			/* Switching to TX after RX on LE 1M/2M PHY */
-			u8_t ppi_en =
-			    HAL_SW_SWITCH_RADIO_ENABLE_S2_PPI(sw_tifs_toggle);
 			u8_t ppi_dis =
 			    HAL_SW_SWITCH_GROUP_TASK_DISABLE_PPI(
 			    sw_tifs_toggle);
 
-			/* Invalidate PPI used when RXing on LE Coded PHY. */
```

Comment:
Reason to remove this invalidation is that the PPI will not be included in the active group.